src/line_range.rs

```diff
@@ -59,15 +62,29 @@ impl LineRange {
 
                 Ok(new_range)
             }
+            3 => {
+                new_range.lower = line_numbers[0].parse()?;
+                new_range.upper = line_numbers[1].parse()?;
+                new_range.increment = Some(line_numbers[2].parse()?);
+                Ok(new_range)
+            }
             _ => Err(
-                "Line range contained more than one ':' character. Expected format: 'N' or 'N:M'"
+                "Line range contained more than one ':' character. Expected format: 'N' or 'N:M' or 'N:M:L'"
```

Comment:
"Line range contained more than one ':' character." needs to be reworded

Reply:
I changed the syntax to `sed` style.
So, I reverted this error message.